sections/collage.liquid

```diff
@@ -46,6 +46,14 @@
                 >
                   {%- if block.settings.image != blank -%}
                     {%- liquid
+                      assign widths = '50, 75, 100, 150, 200, 300, 400, 500, 750, 1000, 1250, 1500, 1750, 2000, 2250, 2500, 2750, 3000, 3200'
+                      assign size_multiplier = 1
+
+                      if block.settings.image_behavior == 'ambient'
+                        assign widths = '60, 90, 120, 180, 240, 360, 480, 600, 900, 1200, 1500, 1800, 2100, 2400, 2700, 3000, 3300, 3600, 3840'
+                        assign size_multiplier = 1.2
```

Comment:
Another general change idea, lmk what you think. Might be nice to also somehow link these multipliers to the value in the css. Or at least document that the 1.2 corresponds to both values. Maybe a comment in the css would suffice. I could certainly see changing that css value to tweak the motion and not remembering to update the multiplier in the sections.

Reply:
Very good point 👍